scripts/testdbsetup.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE lantern_test;
+
+\c lantern_test
+
+\i /docker-entrypoint-initdb.d/dbsetup.sql
+
+\c postgres
```

Comment:
Why does the postgres db need to be re-selected at the end?

Reply:
i liked the idea of setting it back to default. I didn't like the idea of a lingering `lantern_test` connection. It probably isn't necessary, but it was a "just in case" type of thing.